src/Spec-Core.package/SpecInterpreter.class/class/defaultBindings.st

```diff
@@ -1,4 +1,5 @@
 private
 defaultBindings
 	
-	^ MorphicAdapterBindings new
\ No newline at end of file
+	^ (CurrentSpecDefaultBindings value ifNil: [ self environment at: #MorphicAdapterBindings]) new
```

Comment:
the dynamic variable CurrentSpecDefaultBindings now opens possibility to use several Spec bindings in the same image. The fallback to Morphic will be removed in future. It of course brings some performance penalty.